client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -62,43 +95,136 @@ const CancelPurchaseRefundInformation = ( { purchase, includedDomainPurchase } )
 								mappedDomain: includedDomainPurchase.meta,
 							},
 						}
-					),
-				];
+					)
+				);
 
 				showSupportLink = false;
 			} else if ( includedDomainPurchase && isDomainRegistration( includedDomainPurchase ) ) {
-				text = [
-					i18n.translate(
-						'This plan includes the custom domain, %(domain)s, normally a %(domainCost)s purchase. ' +
-							'The domain will not be removed along with the plan, to avoid any interruptions for your visitors. ',
-						{
-							args: {
-								domain: includedDomainPurchase.meta,
-								domainCost: includedDomainPurchase.priceText,
-							},
-						}
-					),
-					i18n.translate(
-						'You will receive a partial refund of %(refundAmount)s which is %(planCost)s for the plan ' +
-							'minus %(domainCost)s for the domain.',
-						{
-							args: {
-								domainCost: includedDomainPurchase.priceText,
-								planCost: purchase.priceText,
-								refundAmount: purchase.refundText,
-							},
-						}
-					),
-				];
-
+				const planCostText =
+					purchase.currencySymbol + ( purchase.refundAmount + includedDomainPurchase.amount );
 				if ( isRefundable( includedDomainPurchase ) ) {
 					text.push(
 						i18n.translate(
-							'To cancel the domain with the plan and ask for a full refund, ' +
-								'please {{contactLink}}contact support{{/contactLink}}.',
+							'Your plan included the custom domain %(domain)s. You can cancel your domain as well as the plan, but keep ' +
+								'in mind that when you cancel a domain you risk losing it forever, and visitors to your site may ' +
+								'experience difficulties acessing it.',
+							{
+								args: {
+									domain: includedDomainPurchase.meta,
+								},
+							}
+						),
+						i18n.translate( "We'd like to offer you two options to choose from:" ),
+						<FormLabel key="keep_bundled_domain">
+							<FormRadio
+								name="keep_bundled_domain_false"
+								value="keep"
+								checked={ ! cancelBundledDomain }
+								onChange={ onCancelBundledDomainChange }
+							/>
+							<span>
+								{ i18n.translate( 'Cancel the plan, but keep %(domain)s.', {
+									args: {
+										domain: includedDomainPurchase.meta,
+									},
+								} ) }
+								<br />
+								{ i18n.translate(
+									"You'll receive a partial refund of %(refundAmount)s -- the cost of the %(productName)s " +
```

Comment:
The `--` should probably be some fancy dash - which one, I always forget 😁

Reply:
This one: — (http://graphemica.com/%E2%80%94).